Update User_Experience.md (#4)
learning-software-engineering/learning-software-engineering.github.io: Topics/User_Experience.md

- [The Definition of User Experience](https://www.nngroup.com/articles/definition-user-experience/)

### Areas of User Experience

User Experience is not easy to define. There are many areas that contribute to User Experience but there is not one answer. The following are a few examples.

___Button Design___ 
- [Amazon Button](https://medium.com/@cccalibour/how-ux-design-makes-a-difference-amazons-continue-button-901618a8b00e): The design of butt
ons that improves user experience by simplifying process.

___Established Norms___
- [Scrolls On Socials](https://forgeandsmith.com/blog/scrolling-vs-clicking-whats-the-preferred-user-experience/): Users are conditioned for scrolling, and now every new social media app conforms to scrolling.

___Color Theory___
- [Colors Influence Choice](https://usabilitygeek.com/colour-user-experience-psychology/#:~:text=Colour%20plays%20a%20crucial%20role,and%20identified%20with%20your%20industry.): Users are shown certain colors to enforce actions. For example, red is commonly linked to aggressive or bad emotions and as such can be used as a cancel to dissuade people from refunding items.

### Helpful Courses

> There are many courses related to User Experience. This includes the U of T Course [The Design of Interactive Computational Media](https://artsci.calendar.utoronto.ca/course/csc318h1) which is highly reccomended if interested in this topic. [The Design of Interactive Computational Media](https://artsci.calendar.utoronto.ca/course/csc318h1) expands on the work done before coding projects. For example, one of the projects is testing how users would interact with a prototype of a UI and modifying it so that the UX is better for the user.

Reviewer: ```suggestion
There are many courses related to User Experience. This includes the U of T Course [The Design of Interactive Computational Media](https://artsci.calendar.utoronto.ca/course/csc318h1) which is highly recommended if interested in this topic. [The Design of Interactive Computational Media](https://artsci.calendar.utoronto.ca/course/csc318h1) expands on the work done before coding projects. For example, one of the projects is testing how users would interact with a prototype of a UI and modifying it so that the UX is better for the user.
```

## Resources for User Experience domain

### Overview

User experience is a large and broad topic. It covers the overall experience of the user and how they interact with software systems and applications. The confusing part is that UI and UX design, while related can also be totally different. For example, UI could be how the interface is designed where UX is more about how users interact with said design. 

> There are many notable links differentiating the two areas. Here is one of the realted websites. 

Reviewer: ```suggestion
There are many notable links differentiating the two areas. Here is one of the related websites. 
```

- [The Definition of User Experience](https://www.nngroup.com/articles/definition-user-experience/)

### Areas of User Experience

> User expereience is not easy to define. There are many areas that contribute to User Expereince but there is not one answer. The following are a few examples.

Reviewer: ```suggestion
User experience is not easy to define. There are many areas that contribute to User Experience but there is not one answer. The following are a few examples.
```

___Button Design___ 

> - [Amazon Button](https://medium.com/@cccalibour/how-ux-design-makes-a-difference-amazons-continue-button-901618a8b00e): The design of butt
> ons that improves user experience by simplifying process.

Reviewer: ```suggestion
- [Amazon Button](https://medium.com/@cccalibour/how-ux-design-makes-a-difference-amazons-continue-button-901618a8b00e): The design of buttons that improves user experience by simplifying process.
```

___Established Norms___
- [Scrolls On Socials](https://forgeandsmith.com/blog/scrolling-vs-clicking-whats-the-preferred-user-experience/): Users are conditioned for scrolling, and now every new social media app conforms to scrolling.

___Color Theory___

> - [Colors Influence Choice](https://usabilitygeek.com/colour-user-experience-psychology/#:~:text=Colour%20plays%20a%20crucial%20role,and%20identified%20with%20your%20industry.): Users are shown certain colors to enforce actions. For example, red is commonly linked to agressive or bad emotions and as such can be used as a cancel to dissuade people from refunding items.

Reviewer: ```suggestion
- [Colors Influence Choice](https://usabilitygeek.com/colour-user-experience-psychology/#:~:text=Colour%20plays%20a%20crucial%20role,and%20identified%20with%20your%20industry.): Users are shown certain colors to enforce actions. For example, red is commonly linked to aggressive or bad emotions and as such can be used as a cancel to dissuade people from refunding items.
```

### Related Resources

> There are many courses related to User Experience. This includes the U of T Course [The Design of Interactive Computational Media](https://artsci.calendar.utoronto.ca/course/csc318h1) which is highly reccomended if interested in this topic.

Reviewer: ```suggestion
There are many courses related to User Experience. This includes the U of T Course [The Design of Interactive Computational Media](https://artsci.calendar.utoronto.ca/course/csc318h1) which is highly recommended if interested in this topic.
```
